reference_system/src/ros2/executor/autoware_default_singlethreaded.cpp

```diff
@@ -14,12 +14,25 @@
 
 #include "rclcpp/rclcpp.hpp"
 
-#include "reference_system_autoware/reference_system.hpp"
+#include "reference_system/system/systems.hpp"
+#include "reference_system/system_builder.hpp"
 
 int main(int argc, char * argv[])
 {
-  create_and_start_reference_system<
-    rclcpp::executors::StaticSingleThreadedExecutor>(argc, argv);
+  rclcpp::init(argc, argv);
+
+  auto nodes = create_autoware_nodes<RclcppSystem>();
+
+#if 0
```

Comment:
Whoopsie this has to be removed. I commented this snippet with `#if 0 ... #endif` since it caused some problems - if it is working fine this could should be uncommented.